lib/board.rb

```diff
@@ -0,0 +1,39 @@
+require_relative './token'
+
+class Board
+    include Token
+    attr_accessor :size, :positions, :max_positions
+    
+    def initialize(size:)
+        @size = size
+        @max_positions = size * size
+        @positions = (1..max_positions).map {|num| num = empty }
+    end
+
+    def update(position:, token:)
+        @positions[position] = token
+    end
+
+    def is_full?()
```

Comment:
In idiomatic Ruby, we usuallly omit `is` as a prefix in methods that return a boolean `is_full? -> full?` 

Similarly, we omit the parentheses when a method doesn't take any parameters.